packages/app/ios/RNFBApp/RNFBUtilsModule.m

```diff
@@ -57,16 +57,16 @@ + (PHAsset *)fetchAssetForPath:(NSString *)localFilePath {
   if ([localFilePath hasPrefix:@"assets-library://"] || [localFilePath hasPrefix:@"ph://"]) {
     if ([localFilePath hasPrefix:@"assets-library://"]) {
       NSURL *localFile = [[NSURL alloc] initWithString:localFilePath];
-#if TARGET_OS_MACCATALYST
```

Comment:
@andymatuschak or @brunolemos - both of you are interested in this working on mac catalyst, can you confirm this change won't be a regression for you? We're trying to remove the AL APIs entirely and this is an intermediate step, but I'm not sure if the `@available` check is strong enough vs the `#if TARGET_nnn` test

Reply:
Hey Mike, if the catalyst build passes that's good enough for my case. I don't think I use this specific feature related to this code.

Reply:
Okay - that's the hard part, I'm not sure it does - we don't have CI coverage of a catalyst build (yet) and none of us are currently targeting that platform personally, so it's untested as of yet. I have some action items related to that, but @brunolemos if you have any links to good docs on how to target react-native at catalyst that would help me cover it in CI to prove this works and keep it squared away for the future

Reply:
Thanks for checking. This'll be fine!

Reply:
Okay great - thanks a ton for giving it a look @andymatuschak - I'll mark this for merge+release then, and of course if I/we are wrong about it I'll be listening and we'll get it fixed up so catalyst works. Cheers